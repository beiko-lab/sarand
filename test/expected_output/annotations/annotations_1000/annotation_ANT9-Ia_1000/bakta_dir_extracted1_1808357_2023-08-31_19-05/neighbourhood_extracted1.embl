ID   contig_1; ; linear; DNA; ; UNC; 2783 BP.
XX
AC   contig_1;
XX
DE   contig_1, whole genome shotgun sequence
XX
OS   .
OC   .
XX
CC   Annotated with Bakta
CC   Software: v1.8.1
CC   Database: v5.0, light
CC   DOI: 10.1099/mgen.0.000685
CC   URL: github.com/oschwengers/bakta
CC   
CC   ##Genome Annotation Summary:##
CC   Annotation Date                :: 08/31/2023, 19:06:00
CC   CDSs                           ::     4
CC   tRNAs                          ::     0
CC   tmRNAs                         ::     0
CC   rRNAs                          ::     0
CC   ncRNAs                         ::     0
CC   regulatory ncRNAs              ::     0
CC   CRISPR Arrays                  ::     0
CC   oriCs/oriVs                    ::     0
CC   oriTs                          ::     0
CC   gaps                           ::     0
CC   pseudogenes                    ::     0
XX
FH   Key             Location/Qualifiers
FH
FT   source          1..2783
FT                   /mol_type="genomic DNA"
FT   gene            11..466
FT                   /locus_tag="KIGIOP_00005"
FT   CDS             11..466
FT                   /product="hypothetical protein"
FT                   /locus_tag="KIGIOP_00005"
FT                   /protein_id="gnl|Bakta|KIGIOP_00005"
FT                   /translation="MPQHIVQKFLGHESPEMTSRYAHIFDETLKNEFTKFQEKLVTNNG
FT                   DVLDLDEDNEVDDVELQWFKKNINAQVLPNGYCRLPVVAGGCPHANACLDCTHFCTSKQ
FT                   FLPQHEEQLERTEELLAIAKDKQWQRQVETNSRVKERLEQIIGSLTG"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT   gene            473..850
FT                   /locus_tag="KIGIOP_00010"
FT   CDS             473..850
FT                   /product="hypothetical protein"
FT                   /locus_tag="KIGIOP_00010"
FT                   /protein_id="gnl|Bakta|KIGIOP_00010"
FT                   /translation="MDKQVRNTTEIVRLAKQKSKKTREKVDKAISKFSIEGKVINFNSI
FT                   AKEANVSKSWLYKEHDIRQRIESLRERQITANVVSKPKKSSRSEEILIKTLKRRVMELE
FT                   KENKKLQNQIQKLYGDLYNKE"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT   gene            1001..1783
FT                   /locus_tag="KIGIOP_00015"
FT                   /gene="ant(9)-Ia"
FT   CDS             1001..1783
FT                   /db_xref="NCBIProtein:WP_000067268.1"
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_P0A0D0"
FT                   /product="aminoglycoside nucleotidyltransferase ANT(9)-Ia"
FT                   /locus_tag="KIGIOP_00015"
FT                   /protein_id="gnl|Bakta|KIGIOP_00015"
FT                   /translation="MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRIN
FT                   SDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPK
FT                   REFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIR
FT                   RAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLL
FT                   DIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_P0A0D0"
FT                   /gene="ant(9)-Ia"
FT   gene            complement(1909..2640)
FT                   /locus_tag="KIGIOP_00020"
FT                   /gene="erm(A)"
FT   CDS             complement(1909..2640)
FT                   /db_xref="NCBIProtein:WP_001072201.1"
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_P13956"
FT                   /product="23S rRNA (adenine(2058)-N(6))-methyltransferase
FT                   Erm(A)"
FT                   /locus_tag="KIGIOP_00020"
FT                   /protein_id="gnl|Bakta|KIGIOP_00020"
FT                   /translation="MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFT
FT                   KELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPY
FT                   NISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYF
FT                   HPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVT
FT                   NINKLSKEQFLSIFNSYKLFH"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_P13956"
FT                   /gene="erm(A)"
XX
SQ   Sequence 2783 BP; 975 A; 417 C; 518 G; 873 T; 0 other;
     taacaacggg atgccccagc atattgtgca gaaatttttg gggcatgaaa gcccagaaat        60
     gacaagcaga tacgctcata tctttgatga aactctaaaa aatgaattta ctaaatttca       120
     ggaaaaactg gttaccaata atggagatgt gcttgatcta gatgaagata atgaagtcga       180
     tgatgtagag cttcaatggt tcaagaaaaa tataaatgca caagtgcttc caaatggtta       240
     ttgtagattg ccagtagtag caggtggttg tccacatgcg aatgcatgct tagattgcac       300
     tcacttctgt accagtaagc aattcttacc acagcacgaa gaacagttag agcgtacaga       360
     agagttatta gccatagcta aggataaaca atggcaaaga caagtagaga ctaatagccg       420
     tgttaaagag cgtttagaac aaatcattgg aagtttgacg gggtaattat caatggataa       480
     acaagttaga aatacaacag aaattgtacg tttggcgaag cagaaatcaa aaaagacaag       540
     ggaaaaagta gacaaagcga tttctaaatt ttcgattgaa ggtaaagtta ttaattttaa       600
     ttcaatagca aaggaagcta atgtttctaa atcatggctt tataaggaac acgatattag       660
     gcaaagaatc gaatcccttc gtgagcgtca aataacagca aatgtagtct caaaacccaa       720
     gaaaagttct cgttcggagg aaatccttat taaaacctta aaaagaagag taatggaatt       780
     agaaaaagaa aataaaaaat tacagaacca aattcaaaaa ttatatggag atctgtataa       840
     taaagaataa ttattaatct gtagacaaat tgtgaaagga tgtacttaaa cgctaacggt       900
     cagctttatt gaacagtaat ttaagtatat gtccaatcta gggtaagtaa attgagtatc       960
     aatataaact ttatatgaac ataatcaacg aggtgaaatc atgagcaatt tgattaacgg      1020
     aaaaatacca aatcaagcga ttcaaacatt aaaaatcgta aaagatttat ttggaagttc      1080
     aatagttgga gtatatctat ttggttcagc agtaaatggt ggtttacgca ttaacagcga      1140
     tgtagatgtt ctagtcgtcg tgaatcatag tttacctcaa ttaactcgaa aaaaactaac      1200
     agaaagacta atgactatat caggaaagat tggaaatacg gattctgtta gaccacttga      1260
     agttacggtt ataaatagga gtgaagttgt cccttggcaa tatcctccaa aaagagaatt      1320
     tatatacggt gagtggctca ggggtgaatt tgagaatgga caaattcagg aaccaagcta      1380
     tgatcctgat ttggctattg ttttagcaca agcaagaaag aatagtattt ctctatttgg      1440
     tcctgattct tcaagtatac ttgtctccgt acctttgaca gatattcgaa gagcaattaa      1500
     ggattctttg ccagaactaa ttgaggggat aaaaggtgat gagcgtaatg taattttaac      1560
     cctagctcga atgtggcaaa cagtgactac tggtgaaatt acctcgaaag atgtcgctgc      1620
     agaatgggct atacctcttt tacctaaaga gcatgtaact ttactggata tagctagaaa      1680
     aggctatcgg ggagagtgtg atgataagtg ggaaggacta tattcaaagg tgaaagcact      1740
     cgttaagtat atgaaaaatt ctatagaaac ttctctcaat taggctaatt ttattgcaat      1800
     aacaggtgct tacttttaaa actactgatt tattgataaa tattgaacaa tttttgggaa      1860
     gaataaagcg tcctcttgtg aaattagaga acgctttatt actttaattt agtgaaacaa      1920
     tttgtaacta ttgaaaatag aaagaaattg ttccttcgat agtttattaa tattagtgac      1980
     atttgcatgc ttcaaagcct gtcggaattg gtttttagtg aaaagaacac gatattcacg      2040
     gtttacccac ttataaacaa aagatcgata ctttttgtag tccttctttg aaatcaatgg      2100
     ttgatgtcgt tcaagaacaa tcaatacaga gtctacactt ggcttaggat gaaaatatag      2160
     tggtggtact tttttgagca tttttatatc catctccacc attaatagta aacccaaagc      2220
     tcgttgcaga ttttgcaatc ttttcgcaaa tcccttctca acgataagat agctatattt      2280
     agcctgactt tcaaaggtaa ttcttttgac aatatccgta ctgatgttat aaggaatatt      2340
     accatatatc ttatagttta tatgttttgg gaaggaaaat tttagaatat ccgtttgaat      2400
     cacttttata ttctcagagg ggtttaccgc ttctttagtc acttgacata agcctccatc      2460
     aatttctata gcagtaactg atcgactcat tttgactagc tctttggtaa aatgtccttt      2520
     tcctgatccg atttctatta cgttgtcttg tttactgata ttcgtgtgat tcaatatttc      2580
     ttttacatgc tttttagaag taataaaatt ttgcgtgtct ttagggtttt tctggttcat      2640
     tataaccttc tccttactgg ttcattataa ccttctcctt actggttata atgaactgac      2700
     attaagcgtt cattataacc aattaatttt gatttggttg ataacgaact ttattaataa      2760
     caaatataga aaaagtaccc ata                                              2783
//
