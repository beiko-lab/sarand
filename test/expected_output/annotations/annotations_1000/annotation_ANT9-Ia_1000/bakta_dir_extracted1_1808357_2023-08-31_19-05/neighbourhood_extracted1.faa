>KIGIOP_00005 hypothetical protein
MPQHIVQKFLGHESPEMTSRYAHIFDETLKNEFTKFQEKLVTNNGDVLDLDEDNEVDDVELQWFKKNINAQVLPNGYCRLPVVAGGCPHANACLDCTHFCTSKQFLPQHEEQLERTEELLAIAKDKQWQRQVETNSRVKERLEQIIGSLTG
>KIGIOP_00010 hypothetical protein
MDKQVRNTTEIVRLAKQKSKKTREKVDKAISKFSIEGKVINFNSIAKEANVSKSWLYKEHDIRQRIESLRERQITANVVSKPKKSSRSEEILIKTLKRRVMELEKENKKLQNQIQKLYGDLYNKE
>KIGIOP_00015 aminoglycoside nucleotidyltransferase ANT(9)-Ia
MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRINSDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPKREFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIRRAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLLDIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN
>KIGIOP_00020 23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(A)
MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFTKELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPYNISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYFHPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVTNINKLSKEQFLSIFNSYKLFH
