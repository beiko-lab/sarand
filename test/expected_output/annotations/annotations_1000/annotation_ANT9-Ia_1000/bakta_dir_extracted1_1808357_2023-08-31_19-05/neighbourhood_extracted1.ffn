>KIGIOP_00005 hypothetical protein
ATGCCCCAGCATATTGTGCAGAAATTTTTGGGGCATGAAAGCCCAGAAATGACAAGCAGATACGCTCATATCTTTGATGAAACTCTAAAAAATGAATTTACTAAATTTCAGGAAAAACTGGTTACCAATAATGGAGATGTGCTTGATCTAGATGAAGATAATGAAGTCGATGATGTAGAGCTTCAATGGTTCAAGAAAAATATAAATGCACAAGTGCTTCCAAATGGTTATTGTAGATTGCCAGTAGTAGCAGGTGGTTGTCCACATGCGAATGCATGCTTAGATTGCACTCACTTCTGTACCAGTAAGCAATTCTTACCACAGCACGAAGAACAGTTAGAGCGTACAGAAGAGTTATTAGCCATAGCTAAGGATAAACAATGGCAAAGACAAGTAGAGACTAATAGCCGTGTTAAAGAGCGTTTAGAACAAATCATTGGAAGTTTGACGGGGTAA
>KIGIOP_00010 hypothetical protein
ATGGATAAACAAGTTAGAAATACAACAGAAATTGTACGTTTGGCGAAGCAGAAATCAAAAAAGACAAGGGAAAAAGTAGACAAAGCGATTTCTAAATTTTCGATTGAAGGTAAAGTTATTAATTTTAATTCAATAGCAAAGGAAGCTAATGTTTCTAAATCATGGCTTTATAAGGAACACGATATTAGGCAAAGAATCGAATCCCTTCGTGAGCGTCAAATAACAGCAAATGTAGTCTCAAAACCCAAGAAAAGTTCTCGTTCGGAGGAAATCCTTATTAAAACCTTAAAAAGAAGAGTAATGGAATTAGAAAAAGAAAATAAAAAATTACAGAACCAAATTCAAAAATTATATGGAGATCTGTATAATAAAGAATAA
>KIGIOP_00015 aminoglycoside nucleotidyltransferase ANT(9)-Ia
ATGAGCAATTTGATTAACGGAAAAATACCAAATCAAGCGATTCAAACATTAAAAATCGTAAAAGATTTATTTGGAAGTTCAATAGTTGGAGTATATCTATTTGGTTCAGCAGTAAATGGTGGTTTACGCATTAACAGCGATGTAGATGTTCTAGTCGTCGTGAATCATAGTTTACCTCAATTAACTCGAAAAAAACTAACAGAAAGACTAATGACTATATCAGGAAAGATTGGAAATACGGATTCTGTTAGACCACTTGAAGTTACGGTTATAAATAGGAGTGAAGTTGTCCCTTGGCAATATCCTCCAAAAAGAGAATTTATATACGGTGAGTGGCTCAGGGGTGAATTTGAGAATGGACAAATTCAGGAACCAAGCTATGATCCTGATTTGGCTATTGTTTTAGCACAAGCAAGAAAGAATAGTATTTCTCTATTTGGTCCTGATTCTTCAAGTATACTTGTCTCCGTACCTTTGACAGATATTCGAAGAGCAATTAAGGATTCTTTGCCAGAACTAATTGAGGGGATAAAAGGTGATGAGCGTAATGTAATTTTAACCCTAGCTCGAATGTGGCAAACAGTGACTACTGGTGAAATTACCTCGAAAGATGTCGCTGCAGAATGGGCTATACCTCTTTTACCTAAAGAGCATGTAACTTTACTGGATATAGCTAGAAAAGGCTATCGGGGAGAGTGTGATGATAAGTGGGAAGGACTATATTCAAAGGTGAAAGCACTCGTTAAGTATATGAAAAATTCTATAGAAACTTCTCTCAATTAG
>KIGIOP_00020 23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(A)
ATGAACCAGAAAAACCCTAAAGACACGCAAAATTTTATTACTTCTAAAAAGCATGTAAAAGAAATATTGAATCACACGAATATCAGTAAACAAGACAACGTAATAGAAATCGGATCAGGAAAAGGACATTTTACCAAAGAGCTAGTCAAAATGAGTCGATCAGTTACTGCTATAGAAATTGATGGAGGCTTATGTCAAGTGACTAAAGAAGCGGTAAACCCCTCTGAGAATATAAAAGTGATTCAAACGGATATTCTAAAATTTTCCTTCCCAAAACATATAAACTATAAGATATATGGTAATATTCCTTATAACATCAGTACGGATATTGTCAAAAGAATTACCTTTGAAAGTCAGGCTAAATATAGCTATCTTATCGTTGAGAAGGGATTTGCGAAAAGATTGCAAAATCTGCAACGAGCTTTGGGTTTACTATTAATGGTGGAGATGGATATAAAAATGCTCAAAAAAGTACCACCACTATATTTTCATCCTAAGCCAAGTGTAGACTCTGTATTGATTGTTCTTGAACGACATCAACCATTGATTTCAAAGAAGGACTACAAAAAGTATCGATCTTTTGTTTATAAGTGGGTAAACCGTGAATATCGTGTTCTTTTCACTAAAAACCAATTCCGACAGGCTTTGAAGCATGCAAATGTCACTAATATTAATAAACTATCGAAGGAACAATTTCTTTCTATTTTCAATAGTTACAAATTGTTTCACTAA
