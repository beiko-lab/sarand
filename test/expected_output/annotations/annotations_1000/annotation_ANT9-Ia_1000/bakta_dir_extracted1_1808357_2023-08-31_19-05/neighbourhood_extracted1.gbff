LOCUS       contig_1                2783 bp    DNA     linear   UNK 31-AUG-2023
DEFINITION  contig_1, whole genome shotgun sequence.
ACCESSION   contig_1
VERSION     contig_1
KEYWORDS    .
SOURCE      None
  ORGANISM  .
            .
COMMENT     Annotated with Bakta
            Software: v1.8.1
            Database: v5.0, light
            DOI: 10.1099/mgen.0.000685
            URL: github.com/oschwengers/bakta
            
            ##Genome Annotation Summary:##
            Annotation Date                :: 08/31/2023, 19:06:00
            CDSs                           ::     4
            tRNAs                          ::     0
            tmRNAs                         ::     0
            rRNAs                          ::     0
            ncRNAs                         ::     0
            regulatory ncRNAs              ::     0
            CRISPR Arrays                  ::     0
            oriCs/oriVs                    ::     0
            oriTs                          ::     0
            gaps                           ::     0
            pseudogenes                    ::     0
FEATURES             Location/Qualifiers
     source          1..2783
                     /mol_type="genomic DNA"
     gene            11..466
                     /locus_tag="KIGIOP_00005"
     CDS             11..466
                     /product="hypothetical protein"
                     /locus_tag="KIGIOP_00005"
                     /protein_id="gnl|Bakta|KIGIOP_00005"
                     /translation="MPQHIVQKFLGHESPEMTSRYAHIFDETLKNEFTKFQEKLVTNNG
                     DVLDLDEDNEVDDVELQWFKKNINAQVLPNGYCRLPVVAGGCPHANACLDCTHFCTSKQ
                     FLPQHEEQLERTEELLAIAKDKQWQRQVETNSRVKERLEQIIGSLTG"
                     /codon_start=1
                     /transl_table=11
                     /inference="ab initio prediction:Prodigal:2.6"
     gene            473..850
                     /locus_tag="KIGIOP_00010"
     CDS             473..850
                     /product="hypothetical protein"
                     /locus_tag="KIGIOP_00010"
                     /protein_id="gnl|Bakta|KIGIOP_00010"
                     /translation="MDKQVRNTTEIVRLAKQKSKKTREKVDKAISKFSIEGKVINFNSI
                     AKEANVSKSWLYKEHDIRQRIESLRERQITANVVSKPKKSSRSEEILIKTLKRRVMELE
                     KENKKLQNQIQKLYGDLYNKE"
                     /codon_start=1
                     /transl_table=11
                     /inference="ab initio prediction:Prodigal:2.6"
     gene            1001..1783
                     /locus_tag="KIGIOP_00015"
                     /gene="ant(9)-Ia"
     CDS             1001..1783
                     /db_xref="NCBIProtein:WP_000067268.1"
                     /db_xref="SO:0001217"
                     /db_xref="UniRef:UniRef50_P0A0D0"
                     /product="aminoglycoside nucleotidyltransferase ANT(9)-Ia"
                     /locus_tag="KIGIOP_00015"
                     /protein_id="gnl|Bakta|KIGIOP_00015"
                     /translation="MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRIN
                     SDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPK
                     REFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIR
                     RAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLL
                     DIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN"
                     /codon_start=1
                     /transl_table=11
                     /inference="ab initio prediction:Prodigal:2.6"
                     /inference="similar to AA sequence:UniRef:UniRef50_P0A0D0"
                     /gene="ant(9)-Ia"
     gene            complement(1909..2640)
                     /locus_tag="KIGIOP_00020"
                     /gene="erm(A)"
     CDS             complement(1909..2640)
                     /db_xref="NCBIProtein:WP_001072201.1"
                     /db_xref="SO:0001217"
                     /db_xref="UniRef:UniRef50_P13956"
                     /product="23S rRNA (adenine(2058)-N(6))-methyltransferase
                     Erm(A)"
                     /locus_tag="KIGIOP_00020"
                     /protein_id="gnl|Bakta|KIGIOP_00020"
                     /translation="MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFT
                     KELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPY
                     NISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYF
                     HPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVT
                     NINKLSKEQFLSIFNSYKLFH"
                     /codon_start=1
                     /transl_table=11
                     /inference="ab initio prediction:Prodigal:2.6"
                     /inference="similar to AA sequence:UniRef:UniRef50_P13956"
                     /gene="erm(A)"
ORIGIN
        1 taacaacggg atgccccagc atattgtgca gaaatttttg gggcatgaaa gcccagaaat
       61 gacaagcaga tacgctcata tctttgatga aactctaaaa aatgaattta ctaaatttca
      121 ggaaaaactg gttaccaata atggagatgt gcttgatcta gatgaagata atgaagtcga
      181 tgatgtagag cttcaatggt tcaagaaaaa tataaatgca caagtgcttc caaatggtta
      241 ttgtagattg ccagtagtag caggtggttg tccacatgcg aatgcatgct tagattgcac
      301 tcacttctgt accagtaagc aattcttacc acagcacgaa gaacagttag agcgtacaga
      361 agagttatta gccatagcta aggataaaca atggcaaaga caagtagaga ctaatagccg
      421 tgttaaagag cgtttagaac aaatcattgg aagtttgacg gggtaattat caatggataa
      481 acaagttaga aatacaacag aaattgtacg tttggcgaag cagaaatcaa aaaagacaag
      541 ggaaaaagta gacaaagcga tttctaaatt ttcgattgaa ggtaaagtta ttaattttaa
      601 ttcaatagca aaggaagcta atgtttctaa atcatggctt tataaggaac acgatattag
      661 gcaaagaatc gaatcccttc gtgagcgtca aataacagca aatgtagtct caaaacccaa
      721 gaaaagttct cgttcggagg aaatccttat taaaacctta aaaagaagag taatggaatt
      781 agaaaaagaa aataaaaaat tacagaacca aattcaaaaa ttatatggag atctgtataa
      841 taaagaataa ttattaatct gtagacaaat tgtgaaagga tgtacttaaa cgctaacggt
      901 cagctttatt gaacagtaat ttaagtatat gtccaatcta gggtaagtaa attgagtatc
      961 aatataaact ttatatgaac ataatcaacg aggtgaaatc atgagcaatt tgattaacgg
     1021 aaaaatacca aatcaagcga ttcaaacatt aaaaatcgta aaagatttat ttggaagttc
     1081 aatagttgga gtatatctat ttggttcagc agtaaatggt ggtttacgca ttaacagcga
     1141 tgtagatgtt ctagtcgtcg tgaatcatag tttacctcaa ttaactcgaa aaaaactaac
     1201 agaaagacta atgactatat caggaaagat tggaaatacg gattctgtta gaccacttga
     1261 agttacggtt ataaatagga gtgaagttgt cccttggcaa tatcctccaa aaagagaatt
     1321 tatatacggt gagtggctca ggggtgaatt tgagaatgga caaattcagg aaccaagcta
     1381 tgatcctgat ttggctattg ttttagcaca agcaagaaag aatagtattt ctctatttgg
     1441 tcctgattct tcaagtatac ttgtctccgt acctttgaca gatattcgaa gagcaattaa
     1501 ggattctttg ccagaactaa ttgaggggat aaaaggtgat gagcgtaatg taattttaac
     1561 cctagctcga atgtggcaaa cagtgactac tggtgaaatt acctcgaaag atgtcgctgc
     1621 agaatgggct atacctcttt tacctaaaga gcatgtaact ttactggata tagctagaaa
     1681 aggctatcgg ggagagtgtg atgataagtg ggaaggacta tattcaaagg tgaaagcact
     1741 cgttaagtat atgaaaaatt ctatagaaac ttctctcaat taggctaatt ttattgcaat
     1801 aacaggtgct tacttttaaa actactgatt tattgataaa tattgaacaa tttttgggaa
     1861 gaataaagcg tcctcttgtg aaattagaga acgctttatt actttaattt agtgaaacaa
     1921 tttgtaacta ttgaaaatag aaagaaattg ttccttcgat agtttattaa tattagtgac
     1981 atttgcatgc ttcaaagcct gtcggaattg gtttttagtg aaaagaacac gatattcacg
     2041 gtttacccac ttataaacaa aagatcgata ctttttgtag tccttctttg aaatcaatgg
     2101 ttgatgtcgt tcaagaacaa tcaatacaga gtctacactt ggcttaggat gaaaatatag
     2161 tggtggtact tttttgagca tttttatatc catctccacc attaatagta aacccaaagc
     2221 tcgttgcaga ttttgcaatc ttttcgcaaa tcccttctca acgataagat agctatattt
     2281 agcctgactt tcaaaggtaa ttcttttgac aatatccgta ctgatgttat aaggaatatt
     2341 accatatatc ttatagttta tatgttttgg gaaggaaaat tttagaatat ccgtttgaat
     2401 cacttttata ttctcagagg ggtttaccgc ttctttagtc acttgacata agcctccatc
     2461 aatttctata gcagtaactg atcgactcat tttgactagc tctttggtaa aatgtccttt
     2521 tcctgatccg atttctatta cgttgtcttg tttactgata ttcgtgtgat tcaatatttc
     2581 ttttacatgc tttttagaag taataaaatt ttgcgtgtct ttagggtttt tctggttcat
     2641 tataaccttc tccttactgg ttcattataa ccttctcctt actggttata atgaactgac
     2701 attaagcgtt cattataacc aattaatttt gatttggttg ataacgaact ttattaataa
     2761 caaatataga aaaagtaccc ata
//
