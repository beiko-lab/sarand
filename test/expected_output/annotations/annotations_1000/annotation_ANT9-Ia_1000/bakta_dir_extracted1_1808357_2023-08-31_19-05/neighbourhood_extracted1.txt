Sequence(s):
Length: 2783
Count: 1
GC: 33.6
N50: 2783
N ratio: 0.0
coding density: 84.4

Annotation:
tRNAs: 0
tmRNAs: 0
rRNAs: 0
ncRNAs: 0
ncRNA regions: 0
CRISPR arrays: 0
CDSs: 4
pseudogenes: 0
hypotheticals: 2
signal peptides: 0
sORFs: 0
gaps: 0
oriCs: 0
oriVs: 0
oriTs: 0

Bakta:
Software: v1.8.1
Database: v5.0, light
DOI: 10.1099/mgen.0.000685
URL: github.com/oschwengers/bakta
